client/blocks/cookie-banner/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { CookieBanner, CookieBannerProps } from '@automattic/privacy-toolset';
+import cookie from 'cookie';
+import { noop } from 'lodash';
```

Comment:
We stopped using `_.noop()` in a while. Let's just declare a local empty function and use that instead. See p4TIVU-9Bf-p2 for more details.